docs/howto/getting-started/accessing-cc-rest-api.md

```diff
@@ -0,0 +1,189 @@
+# Accessing the Cleura Cloud REST API
+
+{{brand}} provides a REST API, which you can take advantage of with a
+tool like `curl`. But before you do, you must create a token for the
+current session. For that, you only need to have an
+[account in {{brand}}](/howto/getting-started/create-account).
+
+## Creating a token
+
+If you do not have two-factor authentication (2FA) enabled for your
+account, to create a token, have your username (`your_cc_username`)
+and password (`your_cc_password`) to the {{brand}} handy and type:
+
+```bash
+curl -H "Content-Type: application/json" \
+    -d '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
+    https://rest.cleura.cloud/auth/v1/tokens
+```
+
+Provided you typed your username and password correctly and that there
+were no connection issues to the remote endpoint, you will get a JSON
+object with a string (`token`) that holds your session token:
+
+```json
+{
+  "result": "login_ok",
+  "token": "vahkie7EiDaij7chegaitee2zohsh1oh"
+}
+```
+
+If, on the other hand, you do have 2FA enabled for your account,
+then first request a token as you would without 2FA enabled:
+
+```bash
+curl -H "Content-Type: application/json" \
+    -d '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
+    https://rest.cleura.cloud/auth/v1/tokens
+```
+
+Instead of a token, you will get a verification code
+(look at `verification`):
+
+```json
+{
+  "result": "twofactor_required",
+  "type": "sms",
+  "verification": "ahb4en3cho"
+}
+```
+
+This verification code is required for requesting a 2FA code:
+
+```bash
+curl -X POST \
```

Comment:
`-X POST` again. This is redundant. :)

Also, do we not have to set a `Content-Type` header here?

Reply:
Ah, yes, `-X POST` is redundand. And I accidentally discovered that we do not have to set a `Content-Type` header, anywhere.